sonar-project.properties

```diff
@@ -0,0 +1,12 @@
+sonar.projectKey=agalit1_capstone-project-url-shortener-frontend
+sonar.organization=agalit1
+
+# This is the name and version displayed in the SonarCloud UI.
```

Comment:
just delete the comments 😄 and you're good to go 💪🏻

Reply:
solved :D thank you!